examples/run.py

```diff
@@ -182,26 +185,28 @@ def fit_age_mp_location(location: str, dmanager: DataManager) -> Dict[str, pd.Da
                                                                   "sex": ["all"]})
     df_all_age_all_sex["death_rate_covid"] = df_all_age_all_sex["deaths_covid"]/df_all_age_all_sex["population"]
     death_rate_covid = df_all_age_all_sex[["time", "death_rate_covid"]].copy()
-    
+
     data_0 = {"0 to 125, all": dmanager.truncate_time_location(location, df_all_age_all_sex, time_end_id=0)}
     data_1 = {"0 to 125, all": dmanager.truncate_time_location(location, df_all_age_all_sex, time_end_id=1)}
     for sex in ["male", "female"]:
-      name = f"0 to 125, {sex}"
-      df = dmanager.read_data_location(location, group_specs={"age_name": ["0 to 125"], "sex": [sex]})
-      df = df.merge(death_rate_covid, on="time")
-      df["deaths_covid"] = df["death_rate_covid"]*df["population"]
-      data_0[name] = dmanager.truncate_time_location(location, df, time_end_id=0)
-      data_1[name] = dmanager.truncate_time_location(location, df, time_end_id=1)
-   
-    for age_group in dmanager.meta[location]["age_groups"]:
-      for sex in ["all", "male", "female"]:
-        name = f"{age_group}, {sex}"
-        df = dmanager.read_data_location(location, group_specs={"age_name": [age_group], "sex": [sex]})
+        name = f"0 to 125, {sex}"
+        df = dmanager.read_data_location(location, group_specs={"age_name": ["0 to 125"], "sex": [sex]})
         df = df.merge(death_rate_covid, on="time")
         df["deaths_covid"] = df["death_rate_covid"]*df["population"]
         data_0[name] = dmanager.truncate_time_location(location, df, time_end_id=0)
         data_1[name] = dmanager.truncate_time_location(location, df, time_end_id=1)
-    models = get_model_mp(data_0)
+
+    for age_group in dmanager.meta[location]["age_groups"]:
+        for sex in ["all", "male", "female"]:
+            name = f"{age_group}, {sex}"
+            df = dmanager.read_data_location(location, group_specs={"age_name": [age_group], "sex": [sex]})
+            df = df.merge(death_rate_covid, on="time")
+            df["deaths_covid"] = df["death_rate_covid"]*df["population"]
+            data_0[name] = dmanager.truncate_time_location(location, df, time_end_id=0)
+            data_1[name] = dmanager.truncate_time_location(location, df, time_end_id=1)
+    models = get_model_mp(data_0,
+                          units_per_year=dmanager.meta[location]["time_start"].units_per_year,
```

Comment:
This line was throwing me an error about not finding the `units_per_year` attribute. For now I've hard-coded `units_per_year=12` since we're modeling monthly, but this might be something to take a look at since we do want to maintain flexibility.

Reply:
```
Traceback (most recent call last):
File "/ihme/code/demographics/krpaul/model_covid_em//02b_fit_mscm_model.py", line 279, in <module>
data_age_mp = fit_age_mp(dmanager)
File "/ihme/code/demographics/krpaul/model_covid_em//02b_fit_mscm_model.py", line 238, in fit_age_mp
results[location] = fit_age_mp_location(location, dmanager)
File "/ihme/code/demographics/krpaul/model_covid_em//02b_fit_mscm_model.py", line 230, in fit_age_mp_location
units_per_year=dmanager.meta[location]["time_start"].units_per_year,
AttributeError: 'list' object has no attribute 'units_per_year'
```

Reply:
Hi Katie, I create a new class called `YearTime` so that it is easier to compute the differences and time axis for the regression. You need to install the version in the `develop` branch to use it.